media/webview.css

```diff
@@ -91,22 +91,3 @@ COL, COLGROUP, TD, TH, CAPTION
   PRE           { page-break-inside: avoid }
   UL, OL, DL    { page-break-before: avoid }
 }
-
-.didact {
-  cursor:crosshair;
-}
-
-.imageRight {
-  float: right;
-  width:33%;
-  height: auto;
-  margin-top:15px;
-  margin-bottom:15px;
-  margin-left:15px;
-}
-
-.imageCenter {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-}
```

Comment:
this has been removed on purpose? I don't see the direct link with theming.

Reply:
This was removed on purpose. It was repeated in the webviewslim.css, which is now shared for both  markdown and asciidoc. Moving them to the *slim.css made more sense since they were there anyway.